model/pail_storage_test.go

```diff
@@ -14,6 +16,9 @@ import (
 )
 
 func TestGetDownloadURL(t *testing.T) {
+	if skip, _ := strconv.ParseBool(os.Getenv("SKIP_INTEGRATION_TESTS")); skip {
```

Comment:
This will still run in the waterfall, correct?

Reply:
Yeah, this is just a convenience flag you can set in your local environment to skip these tests.